packages/sit-onyx/src/components/OnyxButton/types.ts

```diff
@@ -0,0 +1,31 @@
+export type ButtonProps = {
+  /**
+   * The text content of the button.
+   */
+  label: string;
+  /**
+   * If the button should be disabled or not.
+   */
+  disabled?: boolean;
+  /**
+   * The button type.
+   */
+  type?: ButtonType;
+  /**
+   * The color of the button.
+   */
+  color?: ButtonColor;
```

Comment:
```suggestion
  variation?: ButtonVariation;
```
In the figma design this prop is called `variation`, we should align our naming here